webapp/actions/websocket_actions.jsx

```diff
@@ -132,7 +132,7 @@ function handleFirstConnect() {
 
 function handleClose(failCount) {
     if (failCount > MAX_WEBSOCKET_FAILS) {
-        ErrorStore.storeLastError({message: Utils.localizeMessage('channel_loader.socketError', 'Please check connection, Mattermost unreachable. If issue persists, ask administrator to check WebSocket port.')});
+        ErrorStore.storeLastError({message: Utils.localizeMessage('channel_loader.socketError', 'Please check connection, Mattermost unreachable. If issue persists, ask administrator to <a href="https://about.mattermost.com/default-websocket-port-help" target="_blank">check WebSocket port.</a>')});
```

Comment:
This is going to render raw HTML as text in the error message instead of as a link. What you'll want to do instead is create a new [ErrorBarType constant](https://github.com/mattermost/platform/blob/master/webapp/utils/constants.jsx#L290), set that as the message here, and then have the [ErrorBar component](https://github.com/mattermost/platform/blob/master/webapp/components/error_bar.jsx) render it as a `<FormattedHTMLMessage/>` so it actually shows a link